widgets/package.json

```diff
@@ -28,5 +28,8 @@
 		"tslib": "^2.0.0",
 		"typescript": "^4.0.0"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"ngl": "^0.10.4"
```

Comment:
what's the bundle size when adding this client side?

Reply:
according to their repo, >1MB so i don't think we would want to embed that on every page

Reply:
How can we make it separate ?

Isn't the svelte compiler able to make such decisions already based on `import` statements ?